### PR TITLE
Add dynamic links to subsections

### PR DESCRIPTION
**Pull request by i-like-robots**
- Switches out search suggestions API for slurp for relevant results
- Moves styles towards design vision

screen shot 2015-05-18 at 13 55 05

### CODE REVIEW

File: client/components/sub-header/_main.scss [hunk 1]
```diff
@@ -58,21 +79,29 @@
 		padding-bottom: 15px;
 	}
 }
+
+.next-navigation__group__items--columns {
+	@include oGridRespondTo(M) {
+		columns: 2;
```
Comment (ironsidevsquincy) [inline client/components/sub-header/_main.scss]: how does this look in older browsers?

File: client/components/sub-header/_main.scss [hunk 2]
```diff
@@ -58,21 +79,29 @@
...
-			color: #d9d9d9;
+			color: inherit;
 			font-weight: 300;
 
 			&:hover {
 				text-decoration: underline;
 			}
 			&:after {
-				@include nextIcon(arrow-right, #d9d9d9, 10);
+				@include nextIcon(arrow-right, #f99d9d, 10);
```
Comment (ironsidevsquincy) [inline client/components/sub-header/_main.scss]: is this is `next-colors`

**Comment by ironsidevsquincy** [comment]
Awesome :+1:

File: client/components/sub-header/_main.scss (hunk 1 above)
Comment (i-like-robots) [inline client/components/sub-header/_main.scss]: Do we have any hooks - feature or IE specific - to write a fallback against?

File: client/components/sub-header/_main.scss (hunk 2 above)
Comment (i-like-robots) [inline client/components/sub-header/_main.scss]: It's defined here: https://github.com/Financial-Times/next-sass-setup/blob/master/_colors.scss#L12

but `oColorsGetPaletteColor('red-2')` has disastrous consequences :boom:

Comment (ironsidevsquincy) [inline client/components/sub-header/_main.scss]: as in, it doesn't compile?

File: client/components/sub-header/_main.scss (hunk 1 above)
Comment (ironsidevsquincy) [inline client/components/sub-header/_main.scss]: hmmmm, not really. can you write it so it enhances? layout doesn't have to be perfect without `columns`

File: client/components/sub-header/_main.scss (hunk 2 above)
Comment (i-like-robots) [inline client/components/sub-header/_main.scss]: The function returns nothing so `nextIcon` will error out.

Comment (adgad) [inline client/components/sub-header/_main.scss]: I had this problem just now. Moving @import next-sass-setup to the top of client/main.scss should fix it. (I think one of the origami modules was setting up o-colors without our custom defined colours)

Comment (i-like-robots) [inline client/components/sub-header/_main.scss]: thanks @adgad :+1:

File: client/components/sub-header/_main.scss [hunk 3]
```diff
@@ -82,17 +111,18 @@
...
+		padding-right: 20px;
 
-		&:nth-child(-n+3) {
-			margin-top: 0;
-		}
-		&:nth-child(3n+1) {
-			clear: left;
+		// basic columns fallback for IE9
+		float: left\9;
+		width: 50%\9;
```
Comment (i-like-robots) [inline client/components/sub-header/_main.scss]: The best columns fallback I can think without adding feature detection @ironsidevsquincy

File: client/components/sub-header/_main.scss (hunk 1 above)
Comment (i-like-robots) [inline client/components/sub-header/_main.scss]: I guess the difficulty with columns is that it's specified on the parent element and any fallback would be applied to its children. I don't know of a reliable method of getting the same effect :fearful:

File: package.json [hunk 4]
```diff
@@ -24,7 +24,7 @@
     "intl": "^0.1.4",
     "lodash": "^2.4.1",
     "moment": "^2.8.3",
-    "next-ft-api-client": "Financial-Times/next-ft-api-client#v3",
+    "next-ft-api-client": "Financial-Times/next-ft-api-client",
```
Comment (matthew-andrews) [inline package.json]: `^3.0.0`

Comment (matthew-andrews) [inline package.json]: [ it's published on npm now…  sorry i forgot to update ]

**Pull request merged**